android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -572,32 +609,9 @@ public void peerConnectionSetConfiguration(ReadableMap configuration, final int
         peerConnection.setConfiguration(parseRTCConfiguration(configuration));
     }
 
-    String onAddStream(MediaStream mediaStream) {
-        String id = mediaStream.label();
-        String reactTag = null;
-        // The native WebRTC implementation has a special concept of a default
-        // MediaStream instance with the label default that the implementation
-        // reuses.
-        if ("default".equals(id)) {
```

Comment:
I wanted to ask you about that, but I forgot eventually. It is unclear to me how this should be handled. Can we discuss that outside of GH it would take too much time here ?

Reply:
I added it here https://github.com/oney/react-native-webrtc/pull/334/files#diff-8f28661663601f7caa97d6fd5bf1bf30R276